modules/ROOT/pages/security.adoc

```diff
@@ -2,23 +2,15 @@
 include::_partials/attributes.adoc[]
 // end {partialsdir}/attributes.adoc[]
 
-= Security Services
+= Making your mobile app secure
```

Comment:
I don't know what the syntax should be but some of the headings in this page use lower case and some use uppercase. Just an observation. Not sure it's relevant to this pr.